AISKU/src/Initialization.ts

```diff
@@ -278,6 +279,7 @@ export class Initialization implements IApplicationInsights {
         extensions.push(_this.properties);
         extensions.push(_this.dependencies);
         extensions.push(_this.appInsights);
+        _this.context = _this.properties.context;
```

Comment:
the context value of properties is only populated during the initialize() call

Reply:
And "this" is not the snippet, so the aisdk referenced from the snippet is not this object so context is undefined in that case. Need to extendthe ApplicationINsightsContainer to copy the post initialization properties back to the snippet.